src/App.tsx

```diff
@@ -20,18 +24,98 @@ class App extends React.Component<{}, State> {
       './img/9.png',
       './img/10.png',
     ],
+    step: 3,
+    frameSize: 3,
+    itemWidth: 130,
+    animationDuration: 1000,
+  };
+
+  handleChange = (property: string, value: number) => {
+    switch (property) {
+      case 'step':
+        return this.setState({ step: value });
+      case 'frameSize':
+        return this.setState({ frameSize: value });
+      case 'itemWidth':
+        return this.setState({ itemWidth: value });
+      default:
+        return this.setState({ animationDuration: value });
+    }
   };
 
   render() {
-    const { images } = this.state;
+    const {
+      images,
+      step,
+      frameSize,
+      itemWidth,
+      animationDuration,
+    } = this.state;
 
     return (
-      <div className="App">
-        {/* eslint-disable-next-line */}
-        <h1>Carousel with {images.length} images</h1>
+      <>
+        <div className="App">
+          <h1>{`Carousel with ${images.length} images`}</h1>
+
+          <Carousel
+            images={images}
+            step={step}
+            frameSize={frameSize}
+            itemWidth={itemWidth}
+            animationDuration={animationDuration}
+          />
+        </div>
+
+        <form action="#" method="get" className="form">
+          <label>
+            <input
+              type="number"
+              name="step"
+              value={step}
+              onChange={(event) => {
+                this.handleChange(event.target.name, +event.target.value);
```

Comment:
```suggestion
                this.handleChange(event.target);
```
а в handleChange вже можна деструктуризувати